roles/zuul/files/jobs/lore.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: lore
+    node: 'ubuntu-xenial'
+
+    builders:
+      - zuul-git-prep
+      - shell: |
+        ./tests/markdownlint-cli-test.sh
+        ./tests/textlint-test.sh
+        ./tests/shellcheck-test.sh
```

Comment:
@rattboi: these scripts are in the lore repository. The tools they are running (markdownlint, shellcheck, and textlint) are not installed in the images, but I am writing a fix for that right now (the same way I fixed it for hoist before; see BonnyCI/hoist#67).